### PR TITLE
Fix a concurrency bug in stress test, plus clean up of e2e tests

### PR DESCRIPTION
**Pull request by juliev0**
Also, updating documentation for running stress test to include the recently added argument 'eb-type'.

Checklist:

* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-events/blob/master/USERS.md).

### CODE REVIEW

File: test/stress/README.md [hunk 1]
```diff
@@ -82,5 +82,5 @@ It could also hook into your existing EventSources and Sensors to give a simple
 report.
 
 ```shell
-go run ./test/stress/main.go --es-name my-sqs-es --sensor-name my-sensor --hard-timeout 5m
+go run ./test/stress/main.go --eb-type jetstream --es-name my-sqs-es --sensor-name my-sensor --hard-timeout 5m
 ```
```
Comment (juliev0) [inline test/stress/README.md:86]: forgot to update this documentation in the earlier PR, so sneaking it in here

File: test/stress/main.go [hunk 2]
```diff
@@ -446,11 +448,13 @@ Or you can terminate it any time by Ctrl + C.
...
+					esLock.RLock()
 					for _, v := range esTimeMap {
 						if v.After(lastEventTime) {
 							lastEventTime = v
 						}
 					}
+					esLock.RUnlock()
 				}
 				if time.Since(lastEventTime).Seconds() > timeout.Seconds() {
 					fmt.Printf("Exited EventSource Pod %s due to no active events in the last %v\n", podName, o.idleTimeout)
```
Comment (juliev0) [inline test/stress/main.go:460]: Looks like there already was a lock, so I just assume the crash occurred due to the read being unprotected.

**Comment by whynowy** [comment]
Could you remove https://github.com/argoproj/argo-events/blob/master/test/util/util.go#L110-L112 in this PR for me? 

Also, update `stable` to `latest` in https://github.com/argoproj/argo-events/blob/master/test/e2e/fixtures/e2e_suite.go#L54 and https://github.com/argoproj/argo-events/blob/master/manifests/base/controller-manager/controller-config.yaml#L30.

**Comment by juliev0** [comment]
> Could you remove https://github.com/argoproj/argo-events/blob/master/test/util/util.go#L110-L112 in this PR for me?
> 
> Also, update `stable` to `latest` in https://github.com/argoproj/argo-events/blob/master/test/e2e/fixtures/e2e_suite.go#L54 and https://github.com/argoproj/argo-events/blob/master/manifests/base/controller-manager/controller-config.yaml#L30.

Okay, done.

**Review by whynowy — APPROVED** [review]
LGTM

**Pull request merged**